# generated by patch-package 6.4.7 on 2021-12-07 16:03:11
#
# command:
#   npx patch-package babel-eslint
#
# declared package:
#   babel-eslint: 10.0.1
#
diff --git a/node_modules/babel-eslint/lib/analyze-scope.js b/node_modules/babel-eslint/lib/analyze-scope.js
index 9b2a40d..aadcf9e 100644
--- a/node_modules/babel-eslint/lib/analyze-scope.js
+++ b/node_modules/babel-eslint/lib/analyze-scope.js
@@ -328,7 +328,6 @@ module.exports = function(ast, parserOptions) {
   };
 
   if (OriginalReferencer._babelEslintPatched) {
-    require("./patch-eslint-scope")(parserOptions);
     return escope.analyze(ast, options);
   }
 
